Makefile

```diff
@@ -183,8 +190,15 @@ test-simple-float: $(simple_tests-float:=.frun)
 ### Conformance Tests
 
 conformance_tests:=$(wildcard tests/wasm-tests/test/core/*.wast)
+unsupported_conformance_tests:=$(patsubst %, tests/wasm-tests/test/core/%, $(shell cat tests/official/unsupported.txt))
+unparseable_conformance_tests:=$(patsubst %, tests/wasm-tests/test/core/%, $(shell cat tests/official/unparseable.txt))
+parseable_conformance_tests:=$(filter-out $(unparseable_conformance_tests), $(conformance_tests))
+supported_conformance_tests:=$(filter-out $(unsupported_conformance_tests), $(parseable_conformance_tests))
+
+test-conformance-parse: $(parseable_conformance_tests:=.parse)
+test-conformance-supported: $(supported_conformance_tests:=.run-term)
```

Comment:
I would also add `test-conformance-unsupported` (but don't add as subtarget to `test-conformance` target), for ease of saying "what should I work on next?" Up to you.

Reply:
Sure, can just be a `cat` of the files keeping track.

Reply:
Won't fix for now.